source/basic.tex

```diff
@@ -965,7 +965,7 @@
   void h(this X&, int);
   void h(int) &&;               // OK, another overload
   void j(this const X&);
-  void j() const&;              // error: redeclaration
+  void j() const &;             // error: redeclaration
```

Comment:
This is the only occurrence in core wording (in an example). Why not check only the library sources?

Reply:
Sure. Done.